phpunit.xml.dist

```diff
@@ -7,27 +7,30 @@
          beStrictAboutTodoAnnotatedTests="true"
          cacheResultFile="build/.phpunit.cache/test-results"
          colors="true"
+         columns="max"
```

Comment:
Does this play nicely with GitHub Actions?

Reply:
It seems in GitHub Actions the display is already at max columns. Not sure how terminal width is computed there.